package/yast2.changes

```diff
@@ -1,4 +1,17 @@
 -------------------------------------------------------------------
+Wed Nov  5 09:33:08 UTC 2014 - mfilka@suse.com
+
+- bnc#900383
+  - ported several fixes for device renaming
```

Comment:
So this item is a summary of the remaining items, isn't it? Then it is confusing to list it on the same level.